packages/openapi-schema/schema.yml

```
@@ -2503,6 +2503,48 @@ paths:
+                $ref: '#/components/schemas/GetCreditUsageResponse'
+  /credit/balance:
+    get:
+      tags:
+        - credit
+      summary: Get credit balance
+      description: Get credit balance
+      operationId: getCreditBalance
+      responses:
+        '200':
+          description: successful operation
+          content: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)